src/main/java/gift/controller/ProductController.java

```diff
@@ -2,7 +2,11 @@
 
 import gift.dto.ProductRequestDto;
 import gift.dto.ProductResponseDto;
+<<<<<<< step2
+import gift.service.ProductService;
+=======
 import gift.service.ProductServiceImpl;
+>>>>>>> seongwon02
```

Comment:
예를 들어 이런식으로 수정하시면 되겠네요ㅎㅎ
다른 부분들도 충돌 병합 흔적 전부 지워주세요~ 에러가 발생하고 있습니다.

```suggestion
import gift.service.ProductService;
```